src/main/java/com/github/therealguru/totemfletching/TotemFletchingConfig.java

```diff
@@ -3,25 +3,122 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
+
+import java.awt.Color;
 
 @ConfigGroup("totem-fletching")
 public interface TotemFletchingConfig extends Config {
+    @ConfigSection(
+            name = "Overlay Settings",
+            description = "Enable or disable individual overlays",
+            position = 0
+    )
+    String sectionOverlays = "sectionOverlays";
+
+    @ConfigItem(
+            keyName = "renderChatboxOptions",
+            name = "Highlight Chatbox Options",
+            description = "Highlights which animals to select in the UI",
+            section = sectionOverlays,
+            position = 0
+    )
+    default boolean renderChatboxOptions() {
+        return true;
+    }
+
+    @ConfigItem(
+            keyName = "renderTotemOverlays",
+            name = "Show Totem Overlays",
+            description = "Draw the overlay over totems",
+            section = sectionOverlays,
+            position = 1
+    )
+    default boolean renderTotemOverlays() {
+        return true;
+    }
+
+    @ConfigItem(
+            keyName = "renderTextOverlays",
+            name = "Show Totem Text Overlays",
+            description = "Draw the text overlay over totems for animals and total points for the site",
+            section = sectionOverlays,
+            position = 2
+    )
+    default boolean renderTextOverlays() {
+        return true;
+    }
+
+    @ConfigItem(
+            keyName = "renderPoints",
+            name = "Show Points Overlay on ",
```

Comment:
Is there a reason it's "Show Points Overlay on " and not just "Show Points Overlay"?